### DISCUSSION TITLE
Target Imported Component

### DISCUSSION THREAD

**Discussion by nathanaelphilip** [emotion-js/emotion#2687] [Q&A]
Is it possible to import a component from another file and use it for targeting? Something like: 

```
import styled from '@emotion/styled';
import { OtherComponent } from '@/components;

const Component = styled.div`
  margin-bottom: 1rem;

  & + ${OtherComponent} {
   margin-top: -1rem;
  }
`;
```

**Comment by Andarist** [comment]
This works with our `@emotion/babel-plugin`

**Reply by nathanaelphilip** [reply]
<img width="643" alt="CleanShot 2022-03-18 at 13 53 33@2x" src="https://user-images.githubusercontent.com/380034/159066463-bad9a2eb-d1cd-4427-ab52-77bb0babaf8f.png">

I think I have that set.

**Reply by nathanaelphilip** [reply]
this is my `.babelrc`

**Reply by Andarist** [reply]
Well, it looks OK. I can't really say more without looking at the actual repro case. Please always try to share a repro case in a runnable form - either by providing a git repository to clone or a codesandbox. OSS maintainers usually can't afford the time to set up the repro, even if exact steps are given.

I'm afraid though that I won't be able to look into this though - I'm almost 99% sure that this is some kind of a configuration/tooling problem and not the problem with Emotion.

**Reply by nathanaelphilip** [reply]
ok, no worries. thank you!
